.circleci/config.yml

```diff
@@ -44,31 +44,31 @@ jobs:
             prohibit-tab-character.sh
             prohibit-trailing-whitespace.sh
 
-  tests_py36:
+  tests_py36_git_1_8_0:
     executor: ubuntu_executor
     steps:
       - checkout
       - run:
           name: Run tests against Python 3.6 and git 1.8.0
           command: PYTHON_VERSION=3.6 GIT_VERSION=1.8.0  bash ci/tox/ci-run.sh # Earliest versions of python/git supported by git-machete
 
-  tests_py37:
+  tests_py37_git_2_7_6:
     executor: ubuntu_executor
     steps:
       - checkout
       - run:
           name: Run tests against Python 3.7 and git 2.7.6
           command: PYTHON_VERSION=3.7 GIT_VERSION=2.7.6  bash ci/tox/ci-run.sh
 
-  tests_py38:
+  tests_py38_git_2_25_0_pep8:
     executor: ubuntu_executor
     steps:
       - checkout
       - run:
-          name: Run tests against Python 3.8 and git 2.25.0
-          command: PYTHON_VERSION=3.8 GIT_VERSION=2.25.0 bash ci/tox/ci-run.sh
+          name: Run tests against Python 3.8 and git 2.25.0 (check PEP8 correctness)
+          command: PYTHON_VERSION=3.8 GIT_VERSION=2.25.0 FLAKE=true bash ci/tox/ci-run.sh
```

Comment:
I'd use `pep8`/`PEP8` name everywhere... the fact that `flake8` is used underneath is an impl detail